src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -0,0 +1,81 @@
+const BinarySocket = require('../../../../base/socket');
+const localize     = require('../../../../../_common/localize').localize;
+const Url          = require('../../../../../_common/url');
+
+const AccountClosure = (() => {
+    const form_selector = '#form_closure';
+    let $textField;
+    
+    const onLoad = () => {
+        $textField = $('#other-reason');
+
+        $(form_selector).on('submit', (event) => {
+            event.preventDefault();
+            submitForm();
+        });
+        $('#closure-container').setVisibility(1);
+        $textField.on('keyup', () => $textField.removeClass('error-field'));
+    };
+
+    const submitForm = () => {
+        const $btn_submit = $(`${form_selector} #btn_submit`);
+        const reason = getReason();
+        if (reason) {
+            $('#closure_loading').setVisibility(1);
+            $btn_submit.attr('disabled', true);
+
+            const data  = { account_closure: 1, reason };
+            BinarySocket.send(data).then((response) => {
+                if (response.error) {
+                    showFormMessage(localize('Sorry, an error occurred while processing your request.'), false);
```

Comment:
why not show response.error.message to show exactly what went wrong? `response.error.message || localize('Sorry, an error occurred while processing your request.')`